src/foundry/foundry.js/audioContainer.d.ts

```diff
@@ -155,3 +169,7 @@ declare class AudioContainer {
    */
   protected _unloadMediaNode(): void;
 }
+
+declare namespace AudioContainer {
+  type LoadStates = ValueOf<typeof AudioContainer['LOAD_STATES']>;
```

Comment:
in `constants.mjs.d.ts`, we now call these the same as the enum variable, so I suggest to do the same here:
```suggestion
  type LOAD_STATES = ValueOf<typeof AudioContainer['LOAD_STATES']>;
```